docs-mslearn/focus/conformance-summary.md

```diff
@@ -0,0 +1,88 @@
+| ListCost is not supported and will be 0 for EA or MCA Marketplace charges, EA or MCA reservation usage.                                                                                 | Join with price sheet dataset.                | ListCost, ListUnitPrice<br>DH2.4.2, NH1-3, LC1.2, LUP3.1               |
+| SkuId is null when a charge does not have a corresponding price in the price sheet, like savings plan unused charges and Marketplace charges.                                           | No action needed.                             | SkuId<br>SkI3.1                                                        |
+| ChargeDescription is null for savings plan unused charges and charges that are not directly associated with a product SKU.                                                              | Replace with computed alternative as desired. | ChargeDescription<br>CD3                                               |
+| PublisherName is null for reservation usage, reservation purchases, and savings plan unused charges.                                                                                    | Replace null with "Microsoft".                | PublisherName<br>PbN2.2                                                |
+| AvailabilityZone is not available in cost datasets and the column will not be present.                                                                                                  | Join with resource details.                   | AvailabilityZone<br>AZ1                                                |
+
+<br>
+
+## Incorrect data
+
+| Issue                                                                                                       | Workaround                                           | Columns / Requirements                                                                                                                                      |
+| ----------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| BillingPeriodEnd has a known bug where the time is "23:59:59.999" when it should be "00:00:00".             | Remove the time element or update it to be midnight. | BillingPeriodEnd<br>BPE2.2                                                                                                                                  |
+| PricingQuantity and ContractedCost use the wrong scale and are incorrect for Enterprise Agreement accounts. | Multiply by x_PricingBlockSize.                      | ContractedCost, PricingQuantity<br>PQ5, CnC3, CnC5, LC3                                                                                                     |
```

Comment:
```suggestion
| PricingQuantity and ContractedCost use the wrong scale and are incorrect for Enterprise Agreement accounts. | Multiply by x_PricingBlockSize.                      | ContractedCost, PricingQuantity<br>PQ5, CnC3, CnC5, LC3                                                                                                     |
| BillingPeriodEnd has a known bug where the time is "23:59:59.999" when it should be "00:00:00".             | Remove the time element or update it to be midnight. | BillingPeriodEnd<br>BPE2.2                                                                                                                                  |
| Date columns all follow the ISO 8601 standard, but do not include seconds (e.g., “2024-01-01T00:00Z”). | Replace “T00:00Z” with “T00:00:00Z”. Do not add “:00” without confirming seconds are not present. This will be resolved in “1.0r2”. | BillingPeriodStart, BillingPeriodEnd, ChargePeriodStart, ChargePeriodEnd, x_BillingExchangeRateDate, x_ServicePeriodEnd, x_ServicePeriodStart<br>DTF5 |
```